### ISSUE TITLE
clear formatting for pytest output

### ISSUE THREAD

**Issue by squarewave24** [pytest-dev/pytest#5501]
is it possible to make the test output a little more readable? 

currently it  outputs a lot of text sequentially for each test which is hard to parse through. 

here's an example of a npm jest output for React. 

https://imgur.com/8gQsfvJ

it's clear which tests were run /succeeded / failed.. 

is it possible to configure pytest on linux to do anything like this?

**Comment by asottile** [comment]
I believe you're looking for the `-r` option

**Comment by asottile** [comment]
```
  -r chars              show extra test summary info as specified by chars:
                        (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed,
                        (p)assed, (P)assed with output, (a)ll except passed
                        (p/P), or (A)ll. (w)arnings are enabled by default
                        (see --disable-warnings).
```
